xbmc/utils/BufferObject.h

```diff
@@ -28,6 +28,7 @@ class CBufferObject : public IBufferObject
    */
   static std::unique_ptr<CBufferObject> GetBufferObject(bool needsCreateBySize);
 
+  virtual bool CreateBufferObject(uint32_t format, uint32_t width, uint32_t height) override = 0;
```

Comment:
Should probably just be
```suggestion
  bool CreateBufferObject(uint32_t format, uint32_t width, uint32_t height) override { return false; }
```